x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/job/config/AnalysisConfig.java

```diff
@@ -194,8 +188,6 @@ public void writeTo(StreamOutput out) throws IOException {
         if (out.getVersion().before(Version.V_6_5_0)) {
             out.writeBoolean(false);
         }
-
-        out.writeBoolean(usePerPartitionNormalization);
```

Comment:
And here we need to check that if we are writing to an older node we write a `false`.